docs/en/rules/Azure.BV.Immutable.md

```diff
@@ -0,0 +1,87 @@
+---
+severity: Important
+pillar: Security
+category: Security design principles
+resource:  Backup Vault
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.BV.Immutable/
+---
+
+# Immutability
+
+## SYNOPSIS
+
+Ensure immutability is configured to protect backup data.
+
+## DESCRIPTION
+
+Immutability is supported for Backup vaults by configuring the Immutable vault setting.
+
+Immutable vault helps protecting backup data by blocking any operations that could lead to loss of recovery points. Additionally, locking the Immutable vault setting makes it irreversible to prevent any malicious actors from disabling immutability and deleting backups.
```

Comment:
```suggestion
Immutable vault helps protecting backup data by blocking any operations that could lead to loss of recovery points.
Additionally, locking the Immutable vault setting makes it irreversible to prevent any malicious actors from disabling immutability and deleting backups.
For example, an malicious attack may attempt to remove data or delete vaults to prevent recovery to a known good state.
```